### PR TITLE
disabled prop of Loading Buttons in StoryBook restricts color type vi…

### PR DESCRIPTION
**Pull request by alphaxek**
Closes https://github.com/nextui-org/nextui/issues/878

📝 Description
This PR removes duplicate `Spinner Button` from `Button->Loading StoryBook`

⛳️ Current behavior (updates)
Duplicate `Spinner Button` in `Button->Loading StoryBook`

🚀 New behavior
No duplicate `Spinner Button` in `Button->Loading StoryBook`

💣 Is this a breaking change (Yes/No): No

📝 Additional Information
#### Creating new PR because earlier PR (https://github.com/nextui-org/nextui/pull/879) is not able to run GitHub Actions even after 2 approvals.

### CODE REVIEW

**Comment by alphaxek** [comment]
Hi @sun0day , can you please review this PR

**Review by sun0day — APPROVED** [review]

**Comment by alphaxek** [comment]
Thank you @sun0day for your time, looks like first-time contributors need approval from maintainer, can you please help me to take this forward to the maintainers.

**Review by erenkulaksiz — APPROVED** [review]

**Review by jrgarciadev — APPROVED** [review]
Thank you! @alphaxek

**Comment by alphaxek** [comment]
thank you @jrgarciadev

**Pull request merged**
